tests/pytests/unit/runners/test_network.py

```diff
@@ -0,0 +1,255 @@
+"""
+Unit tests for Network runner
+"""
+import logging
+
+import pytest
+import salt.runners.network as network
+from tests.support.mock import MagicMock, patch
+
+log = logging.getLogger(__name__)
+
+MAC_LIST = ["08:00:27:82:b2:ca", "52:54:00:ee:eb:e1", "52:54:00:ee:eb:e1"]
```

Comment:
Apologies for not catching this previously, as explained here: https://docs.saltproject.io/en/master/topics/development/tests/index.html#module-global-level-variables this needs to be a pytest fixture.

Reply:
this is what happens when you C & P from other tests as an example :(

Will adjust, but what annoying since other users will look at tests and copy style etc, and run into the same issue.

Reply:
If there are other tests that are using mutable objects as global variables we need to fix them.